### ISSUE TITLE
question: library / package ecosystem?

### ISSUE THREAD

**Issue by cakekindel** [grain-lang/grain#646]
Hi there! Love the idea and execution on the language, curious if there are plans in place for a dedicate grain package ecosystem? NPM comes with loads of baggage and it would be cool to see something like `cargo` implemented for its ergonomics and small disk footprint. A bespoke package ecosystem would also open the door for doc-gen like docs.rs!

**Comment by ospencer** [comment]
Yes, absolutely! We have plans for a tool like `cargo` that'll be called `silo`. We really love `cargo`, so it'll definitely be influenced a ton by it 🙂

We've been really busy with all sorts of projects and would love to get started on this, so if anyone has experience with package managers or is just interested in helping out, hit us up on Discord!

**Comment by cakekindel** [comment]
Awesome! Where can I find a link to that discord? And I would also love to document what plans exist and facilitate community involvement if you're open to that. Is the plan for grain to be community-maintained a la rust or committee-maintained a la reason?

**Comment by ospencer** [comment]
You can join the Discord here: discord.gg/grain-lang

Curious to hear your ideas. We're definitely still in the early stages, but for now and the foreseeable future it's all community-driven!
